### ISSUE TITLE
Loss problem when training the conditional denoising U-net for diffusion

### ISSUE THREAD

**Issue by ssj9596** [IanYeung/MGLD-VSR#14]
I am currently training a conditional denoising U-net for a diffusion model, but I am encountering an issue with the loss function. When the batch size is set to 1, the loss value remains at 1 and does not decrease. However, when I set the batch size to 4, the loss value becomes NaN. I would like to know which parameters I should adjust to resolve this issue. Thank you very much!

**Comment by projectersMan** [comment]
Hi , I have the same issue , do you solve the problem yet? 
would you like to share the method to solve it with me?

**Comment by zhangyingbit** [comment]
same issue!

**Comment by Rookienovice** [comment]
我也遇到这个问题了，请问你们是什么解决的

**Comment by Rookienovice** [comment]
我也遇到这个问题了，请问你们是怎么解决的 @zhangyingbit

**Comment by zhangyingbit** [comment]
> 我也遇到这个问题了，请问你们是怎么解决的 [@zhangyingbit](https://github.com/zhangyingbit)

我调大了学习率，loss下降了，后来改了模型结构进行训练又出现这个问题了，现在还没有解决！！！
